docs/src/pages/components/passwordfield/react.mdx

```diff
@@ -279,83 +158,81 @@ The standard `ref` prop will forward to the underlying `input` element, and the
 The following is an example demonstrating use of the `ref` and `showPasswordButtonRef` props:
 
 <Example>
-  <RefExample />
-<ExampleCode>
-
-```tsx file=./examples/refs.tsx
-
-```
-
-</ExampleCode>
+  <PasswordFieldRefExample />
+  <ExampleCode>
+    ```tsx file=./examples/PasswordFieldRefExample.tsx ```
+  </ExampleCode>
 </Example>
 
-## CSS Styling
+## Styling
```

Comment:
One change I made to these files was to rename 'CSS Styling' to just 'Styling' because it talks about CSS as well as style props.